build.py

```diff
@@ -80,6 +80,7 @@ def __env(envdir):
 def env():
     """set up a base virtual environment"""
     _env()
+    _deps()
```

Comment:
Alright, so now we have up to four virtualenvs: the `env` one that we build ourselves, which is _only_ used for tox, and then the `.tox/py{27,34,35}` ones, which contain the dependencies for the project. If I want to interact with the project directly (run `py.test`, `import aspen`), then I need to use one of the tox-managed virtualenvs, not `env`. 👍